yt_dlp/extractor/ard.py

```diff
@@ -630,13 +632,36 @@ def _real_extract(self, url):
     }
   }
 }''' % (client, video_id),
-            }).encode(), headers={
-                'Content-Type': 'application/json'
-            })['data']['playerPage']
+                }).encode(), headers={
+                    'Content-Type': 'application/json'
+                })['data']['playerPage']
+
+            content_id = str_or_none(try_get(
+                player_page, lambda x: x['tracking']['atiCustomVars']['contentId']))
+        except ExtractorError as e:
+            # try to get data from website API endpoint as public gateway failed with 503
+            if isinstance(e.cause, HTTPError) and e.cause.status == 503:
```

Comment:
Inverting this check and raising allows us to early exit:
```suggestion
            if not isinstance(e.cause, HTTPError) or e.cause.status != 503:
                raise e
```

Reply:
Use `raise`, not `raise e`